docs.py

```diff
@@ -105,68 +105,36 @@
 
 # add the mapper file
 with open('./docs/simulation-parameter_mapper.json', 'w') as out_file:
-    json.dump(class_mapper(SimulationParameter), out_file, indent=2)
+    json.dump(class_mapper([SimulationParameter]), out_file, indent=2)
 
 
-# generate Radiance SensorGrid open api schema
-print('Generating Radiance SensorGrid documentation...')
+# generate Radiance Objects open api schema
+print('Generating Radiance Objects documentation...')
 
 external_docs = {
     "description": "OpenAPI Specification with Inheritance",
-    "url": "./sensor-grid_inheritance.json"
+    "url": "./radiance_inheritance.json"
 }
 
 openapi = get_openapi(
-    [SensorGrid],
-    title='Honeybee Radiance SensorGrid Schema',
-    description='This is the documentation for Honeybee SensorGrid schema.',
+    [SensorGrid, View],
+    title='Honeybee Radiance Objects Schema',
```

Comment:
Can we call this `"Radiance Asset" instead of "Radiance Objects".  The word "Objects" is so abstract that I would expect to find modifiers and other things under this.

Along these lines, can we also call the json "radiance-asset.json" to make it clear what it is.

Reply:
Sounds good. I can make this change.